### ISSUE TITLE
Error with sierappy seqreads command

### ISSUE THREAD

**Issue by mbyott** [hivdb/sierra-client#13]
Command: 
`sierrapy --url http://localhost:8111/sierra/rest/graphql seqreads merged.insilico_L001_001.codfreq.gz`

Output:
```
100%|██████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<00:00, 37117.73it/s]Traceback (most recent call last):
  File "/home/cloud/miniconda3/bin/sierrapy", line 33, in <module>
    sys.exit(load_entry_point('sierrapy', 'console_scripts', 'sierrapy')())
  File "/mnt/f/src/sierrapy/python/sierrapy/cmds.py", line 5, in main
    cli(obj={})
  File "/home/cloud/miniconda3/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/cloud/miniconda3/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/cloud/miniconda3/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/cloud/miniconda3/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/cloud/miniconda3/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/cloud/miniconda3/lib/python3.8/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/mnt/f/src/sierrapy/python/sierrapy/commands/seqreads.py", line 245, in seqreads
    fns, client._sequence_reads_analysis(payload, query_text)
  File "/mnt/f/src/sierrapy/python/sierrapy/sierraclient.py", line 154, in _sequence_reads_analysis
    result: Dict[str, Any] = self.execute(
  File "/mnt/f/src/sierrapy/python/sierrapy/sierraclient.py", line 69, in execute
    result: Dict[str, Any] = self.client.execute(
  File "/home/cloud/miniconda3/lib/python3.8/site-packages/gql/client.py", line 403, in execute
    return self.execute_sync(
  File "/home/cloud/miniconda3/lib/python3.8/site-packages/gql/client.py", line 221, in execute_sync
    return session.execute(
  File "/home/cloud/miniconda3/lib/python3.8/site-packages/gql/client.py", line 849, in execute
    result = self._execute(
  File "/home/cloud/miniconda3/lib/python3.8/site-packages/gql/client.py", line 744, in _execute
    self.client.validate(document)
  File "/home/cloud/miniconda3/lib/python3.8/site-packages/gql/client.py", line 149, in validate
    raise validation_errors[0]
graphql.error.graphql_error.GraphQLError: Cannot query field 'minReadDepth' on type 'SequenceReadsAnalysis'.

GraphQL request:16:24
15 | }
16 | availableGenes { name }
   |                        ^
17 | minReadDepth
```

**Comment by Nicolas-Fernandez** [comment]
Hi, I have the same issu with 'default' GraphQL 👍 

Command:
`sierrapy seqreads CK0203042.codfreq.gz`

Output:
```
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<00:00, 15592.21it/s]
Traceback (most recent call last):
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/bin/sierrapy", line 10, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/sierrapy/cmds.py", line 5, in main
    cli(obj={})
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/sierrapy/commands/seqreads.py", line 245, in seqreads
    fns, client._sequence_reads_analysis(payload, query_text)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/sierrapy/sierraclient.py", line 154, in _sequence_reads_analysis
    result: Dict[str, Any] = self.execute(
                             ^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/sierrapy/sierraclient.py", line 69, in execute
    result: Dict[str, Any] = self.client.execute(
                             ^^^^^^^^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/gql/client.py", line 403, in execute
    return self.execute_sync(
           ^^^^^^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/gql/client.py", line 221, in execute_sync
    return session.execute(
           ^^^^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/gql/client.py", line 849, in execute
    result = self._execute(
             ^^^^^^^^^^^^^^
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/gql/client.py", line 744, in _execute
    self.client.validate(document)
  File "/Users/2021nf001/miniconda3/envs/SierraPy_v.0.4.1/lib/python3.11/site-packages/gql/client.py", line 149, in validate
    raise validation_errors[0]
graphql.error.graphql_error.GraphQLError: Cannot query field 'minReadDepth' on type 'SequenceReadsAnalysis'.

GraphQL request:16:24
15 | }
16 | availableGenes { name }
   |                        ^
17 | minReadDepth
```

Thank you for your help :)

**Comment by philiptzou** [comment]
@mbyott @Nicolas-Fernandez
It looks like a bug in the default query. I'll make update later.

**Comment by philiptzou** [comment]
This issue has been resolved in SierraPy 0.4.2.

**Comment by Nicolas-Fernandez** [comment]
Dear Philip, again... Thank a lot ;)
Have a nice day, best :)
Nico
